### ISSUE TITLE
Revision: Number of LFS patients

### ISSUE THREAD

**Issue by sjspielman** [AlexsLemonade/OpenPBTA-manuscript#404]
From this PR/conversation: https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1670#issuecomment-1433910182

We need to check this MS line since it maybe should be N=9, _not_ N=8.
https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/b0fdf6f24fe9c021f22e8b7da348708f68877ee5/content/03.results.md?plain=1#L203

**Comment by jharenza** [comment]
Leaving this here, will reword in text:

```r
> v23 %>%
+   filter(experimental_strategy == "RNA-Seq",
+          grepl("Li", cancer_predispositions)) %>%
+ count(Kids_First_Participant_ID)
# A tibble: 8 × 2
  Kids_First_Participant_ID     n
  <chr>                     <int>
1 PT_89XRZBSG                   2
2 PT_DPVW4M1W                   1
3 PT_ESHACWF6                   2
4 PT_JT9HH7M6                   2
5 PT_PF9EC93Q                   1
6 PT_PFP1ZVHD                   1
7 PT_VZTTTP3Q                   1
8 PT_YZ8A8A36                   1
```

**Comment by jharenza** [comment]
closed with #405
